library/ssl_tls12_client.c

```diff
@@ -1810,11 +1810,7 @@ static int ssl_check_server_ecdh_params(const mbedtls_ssl_context *ssl)
     return 0;
 }
 
-#endif /* MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED   ||
-          MBEDTLS_KEY_EXCHANGE_ECDH_RSA_ENABLED    ||
-          MBEDTLS_KEY_EXCHANGE_ECDHE_PSK_ENABLED   ||
-          MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED ||
-          MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED */
+#endif /* MBEDTLS_KEY_EXCHANGE_ECDH*_ENABLED */
```

Comment:
I didn't get the reason for this change since this comment and the similar one below are removed in the following commit. Can't we remove this?